src/common/resolve.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+let { resolve } = eval("require");
+
+// In the VS Code extension `require` is overridden and `require.resolve` doesn't support the 2nd argument.
+if (resolve.length === 1) {
+  const Module = eval("require")("module");
+  const createRequire = Module.createRequire || Module.createRequireFromPath;
+  resolve = createRequire(__dirname).resolve;
+}
```

Comment:
I guess unnecessary to do another length check.

Reply:
Not sure what you mean.

Reply:
I mean do we need check `resolve.length` again and throw an error if length still `!=2`, but I guess it's unnecessary.

Reply:
Agree.